website/docs/getting-started/setup/upgrade-to-business-edition/upgrade-k8s-to-business-edition.mdx

```diff
@@ -2,9 +2,25 @@
 
 <HideElements/>
 
-Follow the steps below to upgrade Kubernetes Community edition to Business edition:
+Follow the steps below to upgrade the Kubernetes Community edition to the Business edition:
 
 <ul class="circle"> 
+
+<li> Create a backup of the CE data available at <code>/appsmith-stacks/data/backup</code>. Connect to the shell of a running container.
+
+         kubectl exec -it <pod> bash
+    
+    /* Run the backup command */
+    appsmithctl backup
+
```

Comment:
```suggestion

/* Retrieve the salt and password (These values will need to be added to values.yaml file of the appsmith-ee helm deployment) */
    cat /appsmith-stacks/configuration/docker.env | grep APPSMITH_ENCRYPTION_

```

Reply:
Getting the enc values while still in the pod shell

Reply:
@sum35h again the commands look different from the one on the original - https://docs.appsmith.com/getting-started/setup/upgrade-to-business-edition/kubernetes#backup-data

Should we be updating those as well?  

Also this PR was to shorten the steps or skip some as this will be page embedded in the customer support portal

cc: @pranayagarwal96